src/NzbDrone.Api/Calendar/CalendarFeedModule.cs

```diff
@@ -62,6 +67,12 @@ private Response GetCalendarFeed()
                 premiersOnly = bool.Parse(queryPremiersOnly.Value);
             }
 
+            if (queryTags.HasValue)
+            {
+                var tagInput = (string)queryTags.Value.ToString();
+                tags = tagInput.Split(',').Select(_tagService.GetTag).OfType<Tag>().Select(t => t.Id).ToArray();
```

Comment:
The null-check isn't needed anymore either. GetTag throws if a tag doesn't exist.